doc/tutorials/cloud/00_setup.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<p>No, we won't load the notebook, we will guide the user through a series of notebooks by links between them and have in the beginning also a table of contents.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Yes, that's only for development and initial version of notebook functionality. This includes will go away after notebook-connect refactor</p>

Reply:
<p>ok, sounds good</p>